test/data/activity.js

```diff
@@ -0,0 +1,25 @@
+const uuid = require('uuid/v4');
+const moment = require('moment');
+const ids = require('./ids');
+const { asruSuper } = require('./profiles');
+
+const generateDates = daysAgo => {
+  const date = moment().subtract(daysAgo, 'days').toISOString();
+  return {
+    createdAt: date,
+    updatedAt: date
+  };
+};
+
+const activity = [
+  {
+    id: uuid(),
+    caseId: ids.task.project.discardedByAsru,
+    changedBy: asruSuper.id,
+    eventName: 'status:with-inspectorate:discarded-by-asru',
+    event: {},
+    ...generateDates(34)
```

Comment:
not strictly needed but should make the data seem a bit more realistic as it tallies with the dates on the task itself.